raiden/api/python.py

```diff
@@ -115,26 +120,35 @@ def token_network_connect(
         if not isaddress(token_address):
             raise InvalidAddress('token_address must be a valid address in binary')
 
-        connection_manager = self.raiden.connection_manager_for_token(token_address)
+        connection_manager = self.raiden.connection_manager_for_token(
+            registry_address,
+            token_address
```

Comment:
Please, add trailing commas to function arguments (in function signatures with multiline arguments and function calls)